apps/cricket-score-frontend/src/components/match/ErrorDisplay.tsx

```
@@ -0,0 +1,39 @@
+      <div className="w-16 h-16 mx-auto bg-red-500/30 backdrop-blur-sm rounded-full flex items-center justify-center border-2 border-red-500/50 relative shadow-xl">
+        <svg xmlns="http://www.w3.org/2000/svg" className="h-8 w-8 text-red-400" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z" />
+        </svg>
+      </div>
+      <h2 className="mt-4 text-2xl font-bold text-foreground">Connection Error</h2>
+      <p className="mt-2 text-muted-foreground">{error}</p>
+      <div className="mt-6 flex gap-3 justify-center relative">
+        <button
+          onClick={onRetry}
+          className="btn btn-primary px-5 py-2.5"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)